$MirOS: ports/databases/sqlite/patches/patch-src_shell_c,v 1.3 2007/05/06 20:22:31 tg Exp $
--- src/shell.c.orig	Tue Aug  5 19:58:27 2008
+++ src/shell.c	Tue Aug 26 12:22:43 2008
@@ -64,6 +64,10 @@ extern int isatty();
 #include <sys/time.h>
 #include <sys/resource.h>
 
+#include "regexp.h"
+
+__RCSID("$MirOS$");
+
 /* Saved resource information for the beginning of an operation */
 static struct rusage sBegin;
 
@@ -956,6 +960,7 @@ static void open_db(struct callback_data
     sqlite3_open(p->zDbFilename, &p->db);
     db = p->db;
     if( db && sqlite3_errcode(db)==SQLITE_OK ){
+      sqlite3_regexp_posixext_register(db);
       sqlite3_create_function(db, "shellstatic", 0, SQLITE_UTF8, 0,
           shellstaticFunc, 0, 0);
     }
